community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
+
+## Basic Protocol in case you are harassed/bullied/found something offensive/weird/unacceptable
+
+Subject -> Who has been harassed/bullied
+
+Type 1 :- Constitutionally Legal Offense
+
+1. In this case the team is requested not to micromanage. let it go through official channels of college and judiciary and let them deal with it.
+
+Type 2 :- Everything Else
+
+1. If it has happened for the first time, dm the person. And let {pronoun} know that this behaviour is offensive to the subject. If not comfortable confronting then **????????????? @team comments**
+2. Post on a public forum that the subject finds this particular behaviour offensive/xyz. Name calling doesn't help.
+3. If it's repetitive, **what to do ???????????????????? @team , comments**
+
+## Basic Protocol in case you are accused of harassment/bullying/offending/doing weird shits/unacceptable behaviour
+
+Subject -> Who has been accused
+
+1. Well the subject can start by saying sorry, and ensuring that similar behaviour is not repeated in front of the person who has accused the subject "ever". If it's ingrained in the subject’s habit let the accussee know that, it's in your habit you will make sincere effort to decrease the frequency. The subject can take a note and share updates of progress {pronoun} have made on that person's feedback.
+2. Adding "genuinely" prefix in front of sorry doesn't help.
+3. It doesn't matter if the subject’s sorry or change is genuine or fake, as long as it's change it's good.
+
+## Basic Protocol in case you have noticed harassment.
+
+**@team comments**
+
+
+
+## Basic Protocol in case you are an executive/advisor and have to take action
+
+**@team comments**
+
+## What is Considered as Sexual Harassment at Work Place ?
+
+What workplace at koss includes, **@team ????????????**
+
+Sexual harassment includes many things but is not limited to
+
+[Ref](https://www.un.org/womenwatch/osagi/pdf/whatissh.pdf)
+
+1. Actual or attempted rape or sexual assault.
+2. Unwanted pressure for sexual favors.
+3. Unwanted deliberate touching, leaning over, cornering, or pinching.
+4. Unwanted sexual looks or gestures.
+5. Unwanted letters, telephone calls, or materials of a sexual nature.
+6. Unwanted pressure for dates.
+7. Unwanted sexual teasing, jokes, remarks, or questions.
+8. Referring to an adult as a girl, hunk, doll, babe, or honey.
+9. Whistling at someone.
+10. Cat calls.
+11. Sexual comments.
+12. Turning work discussions to sexual topics.
+13. Sexual innuendos or stories.
+14. Asking about sexual fantasies, preferences, or history.
+15. Personal questions about social or sexual life.
+16. Sexual comments about a person's clothing, anatomy, or looks.
+17. Kissing sounds, howling, and smacking lips.
+18. Telling lies or spreading rumors about a person's personal sex life.
+19. Neck massage.
```

Comment:
To be honest, I don't think the victim will ever use the term _neck massage_ to talk about this. The third point already covers this: 3. Unwanted deliberate touching, leaning over, cornering, or pinching.